components/aws/sagemaker/process/README.md

```diff
@@ -13,29 +12,69 @@ Argument        | Description                 | Optional   | Data type  | Accept
 output_encryption_key | The AWS KMS key that Amazon SageMaker uses to encrypt the model artifacts | Yes | String | | |
-max_run_time | The maximum run time in seconds per training job | Yes | Int | ≤ 432000 (5 days) | 86400 (1 day) |
+max_run_time | The maximum run time in seconds per processing job | Yes | Int | ≤ 432000 (5 days) | 86400 (1 day) |
 environment | The environment variables to set in the Docker container | Yes | Yes | Dict | Maximum length of 1024. Key Pattern: `[a-zA-Z_][a-zA-Z0-9_]*`. Value Pattern: `[\S\s]*`. Upto 16 key and values entries in the map | |
-container_entrypoint | The entrypoint for the processing job. This is in the form of a list of strings that make a command | Yes | Yes | List of Dicts | | [] |
-container_arguments | A list of string arguments to be passed to a processing job | Yes | Yes | List of Dicts | | [] |
-input_config | Parameters that specify Amazon S3 inputs for a processing job | No | Dict | | {} |
-output_config | Parameters that specify Amazon S3 outputs for a processing job | No | Dict | | {} |
+container_entrypoint | The entrypoint for the processing job. This is in the form of a list of strings that make a command | Yes | Yes | List of Strings | | [] |
+container_arguments | A list of string arguments to be passed to a processing job | Yes | Yes | List of Strings | | [] |
+input_config | Parameters that specify Amazon S3 inputs for a processing job | No | List of Dicts | | [] |
+output_config | Parameters that specify Amazon S3 outputs for a processing job | No | List of Dict | | [] |
 vpc_security_group_ids | A comma-delimited list of security group IDs, in the form sg-xxxxxxxx | Yes | String | | |
 vpc_subnets | A comma-delimited list of subnet IDs in the VPC to which you want to connect your hpo job | Yes | String | | |
-network_isolation | Isolates the training container if true | No | Boolean | False, True | True |
-traffic_encryption | Encrypts all communications between ML compute instances in distributed training if true | No | Boolean | False, True | False |
+network_isolation | Isolates the processing container if true | No | Boolean | False, True | True |
+traffic_encryption | Encrypts all communications between ML compute instances in distributed processing if true | No | Boolean | False, True | False |
 tags | Key-value pairs to categorize AWS resources | Yes | Dict | | {} |
 
+Notes:
+* You can find more information about how container entrypoint and arguments are used at the [Build Your Own Processing Container](https://docs.aws.amazon.com/sagemaker/latest/dg/build-your-own-processing-container.html#byoc-run-image) documentation.
+* Each key and value in the `environment` parameter string to string map can have length of up to 1024. SageMaker supports up to 16 entries in the map.
+* The format for the `input_config` field is:
```

Comment:
make [input_config](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_ProcessingInput.html) hyperlink as well so that customer can know which fields are mandatory

same for output config